resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -392,6 +392,13 @@ private[spark] object Config extends Logging {
   val KUBERNETES_AUTH_SUBMISSION_CONF_PREFIX =
     "spark.kubernetes.authenticate.submission"
 
+  val KUBERNETES_TRUST_CERTIFICATES =
+    ConfigBuilder("spark.kubernetes.trust.certificates")
+      .doc("If set to true then client can submit to kubernetes cluster only with token")
+      .version("3.2.0")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
For the security, shall we make this `true` by default? For the users who don't have `ca`, they can use `spark.kubernetes.trust.certificates=false` explicitly, @hddong ?

Reply:
> For the security, shall we make this `true` by default? For the users who don't have `ca`, they can use `spark.kubernetes.trust.certificates=false` explicitly, @hddong ?

`false` means need `ca`, so default value is what you expect.

Reply:
I think maybe the confusion indicates we could find a better name for the config.